JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -144,29 +146,27 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
                                          missing = missing, callback = progressbarTick))
           
           relyFit$freq$est$freq_alpha <- Bayesrel:::applyalpha(model[["dat_cov"]])
-          p <- ncol(dataset)
-          Ctmp <- array(0, c(p, p - 1, p - 1))
-          for (i in 1:p){
-            Ctmp[i, , ] <- model[["dat_cov"]][-i, -i]
-          }
+          relyFit[["freq"]][["est"]] <- relyFit[["freq"]][["est"]][c(5, 1, 2, 3, 4)]
+            
+          Ctmp <- .itemDeletedM(model[["dat_cov"]])
+          
           relyFit$freq$ifitem$alpha <- apply(Ctmp, 1, Bayesrel:::applyalpha)
+          relyFit[["freq"]][["ifitem"]] <- relyFit[["freq"]][["ifitem"]][c(5, 1, 2, 3, 4)]
+          
           
           if (!alphaAna) { # when standardized alpha, but bootstrapped alpha interval:
             cors <- array(0, c(options[["noSamples"]], p, p))
             for (i in 1:options[["noSamples"]]) {
               cors[i, , ] <- .cov2cor.callback(relyFit$freq$covsamp[i, , ], progressbarTick)
             }
             relyFit$freq$boot$alpha <- apply(cors, 1, Bayesrel:::applyalpha)
-            if (omegaAna && is.null(relyFit[["freq"]][["omega.error"]])) {
+            if (length(relyFit[["freq"]][["boot"]]) == 4)
               relyFit[["freq"]][["boot"]] <- relyFit[["freq"]][["boot"]][c(4, 1, 2, 3)]
-            } else {
+            else if (length(relyFit[["freq"]][["boot"]]) == 5)
```

Comment:
This is related to the above, is it possible to somehow avoid these magic numbers?

Reply:
I will try to smoothen this out in the future.

Reply:
No, I think this is fine as it is